v3/internal/assetserver/assetserver.go

```diff
@@ -1,22 +1,19 @@
 package assetserver
 
 import (
-	"embed"
 	"fmt"
-	"io/fs"
 	"net"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
-	"path"
 	"strings"
 	"time"
 )
 
 const (
 	webViewRequestHeaderWindowId   = "x-wails-window-id"
 	webViewRequestHeaderWindowName = "x-wails-window-name"
-	pluginPrefix                   = "/wails/plugin"
+	servicePrefix                   = "/services"
```

Comment:
I think we should keep "reserved" paths to a `/wails/` prefix. Thoughts?

Reply:
I have no issue with that

Reply:
Resolved in latest commit